app/assets/javascripts/dashboard.js

```diff
@@ -27,6 +27,16 @@ MinionPoller = {
           }
         }
         $(".nodes-container tbody").html(rendered);
+
+        var unassignedMinions = data.unassigned_minions;
+
+        if (unassignedMinions.length > 0) {
+          $('#unassigned_count').html(unassignedMinions.length + " \
+            <strong>new</strong> nodes are available but have not been added to the cluster yet"
```

Comment:
It is not that simple: 

- **1** new **node** **is** available but *has* not been...
- **2** new **nodes** **are** available but **have** not been...

I don't think it is worth it. This text will probably be gone soon.